tools/update_cls.R

```diff
@@ -0,0 +1,19 @@
+# Helper script for updating .cls file in this package
+
+# JSS article -------------------------------------------------------------
+# From https://www.jstatsoft.org/pages/view/style
+dir.create(tmp_dir <- tempfile())
+bundle <- "https://www.jstatsoft.org/public/journals/1/jss-article-tex.zip"
+new_jss <- xfun::in_dir(tmp_dir, {
+  xfun::download_file(bundle)
+  unzip(basename(bundle))
+  xfun::normalize_path("jss.cls")
+})
+file.copy(new_jss,
```

Comment:
We should probably also copy `jsslogo.jpg` and `jss.bst`, although they are much less likely to change.

Reply:
Oh you're right ! That is obvious and yet I missed it as I was focus on the `jss.cls` 😅

Reply:
Done in 07c5094